microsoft-365/compliance/gdpr-arc-Office365.md

```diff
@@ -88,7 +88,7 @@ This control structure is also used to organize the presentation of the internal
 |***Information Security Policies (6.2)***|The customer should augment any existing information security policies to include protection of personal data, including policies necessary for compliance with any applicable legislation.|Microsoft policies for information security and any specific measures for the protection of personal information.<br>- *Microsoft Office 365 (All-Up) ISO/IEC 27001:2013 ISMS Statement of Applicability* [[12](gdpr-arc-Office365.md#12)] see A.19<br>- *SOC 2 Type 2 Audit Report* [[11](gdpr-arc-Office365.md#11)]<br>- 6.2 Information security policies in [Compliance Manager](https://servicetrust.microsoft.com/ComplianceManager)|24(2)|
-|***Organization of Information Security Customer consideration (6.3)***|The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles.|An overview of the role of Microsoft's Data Protection Officer, the nature of his duties, reporting structure and contact information.<br>- *Microsoft’s Data Protection Officer* [[18](gdpr-arc-Office365.md#18)]<br>- *Office 365 ISMS Manua*l [[14](gdpr-arc-Office365.md#14)] see *5.3 ORGANIZATIONAL ROLES, RESPONSIBILITIES, AND AUTHORITIES*<br>- 6.3 Organization of information security in [Compliance Manager](https://servicetrust.microsoft.com/ComplianceManager)|(37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2)|
+|***Organization of Information Security Customer consideration (6.3)***|The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles.|An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure and contact information.<br>- *Microsoft’s Data Protection Officer* [[18](gdpr-arc-Office365.md#18)]<br>- *Office 365 ISMS Manua*l [[14](gdpr-arc-Office365.md#14)] see *5.3 ORGANIZATIONAL ROLES, RESPONSIBILITIES, AND AUTHORITIES*<br>- 6.3 Organization of information security in [Compliance Manager](https://servicetrust.microsoft.com/ComplianceManager)|(37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2)|
```

Comment:
```suggestion
|***Organization of Information Security Customer consideration (6.3)***|The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles.|An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure, and contact information.<br>- *Microsoft’s Data Protection Officer* [[18](gdpr-arc-Office365.md#18)]<br>- *Office 365 ISMS Manua*l [[14](gdpr-arc-Office365.md#14)] see *5.3 ORGANIZATIONAL ROLES, RESPONSIBILITIES, AND AUTHORITIES*<br>- 6.3 Organization of information security in [Compliance Manager](https://servicetrust.microsoft.com/ComplianceManager)|(37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2)|
```